src/main/java/com/gmail/inverseconduit/bot/DefaultBot.java

```diff
@@ -58,7 +61,10 @@ private void processMessage(final ChatMessage chatMessage) {
         final String trigger = AppContext.INSTANCE.get(BotConfig.class).getTrigger();
         if ( !chatMessage.getMessage().startsWith(trigger)) { return; }
 
-        commands.stream().filter(c -> chatMessage.getMessage().replace(trigger, "").startsWith(c.getName())).findFirst().map(c -> c.execute(chatMessage))
+        final String commandName = chatMessage.getMessage().replace(trigger, "").split(" ")[0];
```

Comment:
Wait, just discovered a problem. Considering that we're using `.replace()` method which replaces all occurrences of a literal sequence, wouldn't this tear away parts of the message if it includes the trigger as well? `!!h!!i` -> `hi` - I think we need `replaceFirst`.